mosquitto.c

```diff
@@ -1147,7 +1147,12 @@ PHP_MOSQUITTO_API void php_mosquitto_message_callback(struct mosquitto *mosq, vo
 
 	object_init_ex(message_zval, mosquitto_ce_message);
 	message_object = mosquitto_message_object_from_zend_object(Z_OBJ_P(message_zval));
-	mosquitto_message_copy(&message_object->message, message);
+	message_object->message.mid = message->mid;
```

Comment:
Yes.
At first i used 2 simple scripts. One for subscribing and second for publishing 100000 messages. I run this scripts and after that watch in htop linux utility at memory column. In htop memory always grow.
After that i run subscribe script in valgrind, i saw the problem. Leak was in PHP5 and PHP7